root.store.js

```diff
@@ -22,7 +31,20 @@ const getEnhancers = () => {
   return enhancers;
 };
 
-export const configureStore = createStore(
-  rootReducer,
-  compose(getMiddleware(), ...getEnhancers())
-);
+
+export default function configureStore() {
+  const store = createStore(rootReducer, compose(getMiddleware(), ...getEnhancers()));
+
+  if (module.hot) {
+    module.hot.accept(() => {
+      store.replaceReducer(require('./root.reducer').default); // eslint-disable-line global-require
+    });
+  }
+
+  if (__DEV__) {
```

Comment:
+1 on that question? What are you trying to do with this? I assume it is related with redux devtools but not 100% sure though.